compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/Symbols.java

```diff
@@ -223,9 +223,7 @@ public static BXMLNSSymbol createXMLNSSymbol(Name name,
     }
 
     public static BVarSymbol createVarSymbolForTupleMember(BType type) {
-        BTypeSymbol tsymbol = type.tsymbol;
-        return new BVarSymbol(type.flags, tsymbol.name, tsymbol.pkgID, type, tsymbol.owner,
-                tsymbol.pos, tsymbol.origin);
+        return new BVarSymbol(type.flags, null, null, type, null, null, null);
```

Comment:
Flags also are technically not those of the type.

Can you point to where the name is set for the tuple members?

Reply:
@ushirask

Reply:
It's done with `visit(BLangSimpleVariable varNode)` in SymbolEnter when going through `transform(BLangTupleTypeNode tupleTypeNode, AnalyzerData data)` in SymbolAnalyzer. We might be missing the name otherwise.

Reply:
As discussed, please fix in a separate PR.

Reply:
We need to properly set symbol origin also.